### PR TITLE
Lab 2 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,8 +46,10 @@ <h3>haiku</h3>
        </main>
     <footer>
         <hr>
-        <blockquote>God is great, beer is good, and people are crazy </blockquote>
-        <cite>&mdash;Author <span class="author">Unknown.</span></cite>
+        <blockquote>God is great, beer is good, and people are crazy 
+            <cite>&mdash;Author <span class="author">Unknown.</span></cite>
+        </blockquote>
+
```
Comment (rmartin-sc) [inline index.html]: Lab instructions were to put `<cite>` inside `<blockquote>`

Comment (stevenmac92) [reply index.html]: Oh sorry I was helped with that, ill fix that.

**Comment by rmartin-sc** [comment]
All you need to do is merge the pull request and it will be fixed!
________________________________

**Comment by rmartin-sc** [comment]
(See the git demo video in LMS)
________________________________

**Pull request merged**